djadmin2/views.py

```diff
@@ -19,17 +21,14 @@
 from django.utils.text import capfirst
 from django.utils.translation import ugettext_lazy
 from django.views import generic
-
 import extra_views
 
 
 from . import permissions, utils
 from .forms import AdminAuthenticationForm
-from .models import LogEntry
```

Comment:
Yes, there is, I don't fully remember but this was related to new way how models are initialized in Django 1.7.